CONTRIBUTING.md

```diff
@@ -140,15 +140,18 @@ The new extension window will appear with window title `Extension development ho
 
 ### Working with FSAC
 
-1. Run FSAC:
-    - or from sources in your favorite editor: open the `FsAutoComplete.sln`, start debugging with  `.NET Core mode http (debug)` configuration (default to port `8088`)
-    - or run it with `--mode http --port 8088` arguments
-2. In vscode, set the `FSharp.fsacUrl` setting to `http://localhost:8088`
-3. Reload the instance with `> Developer: Reload Window`.
-
-The port number can be changed as needed.
-
-Remove that setting to go back to FSAC bundled in Ionide extension.
+1. Open FSAC from a new instance of VSCode from the directory: `paket-files/github.com/fsharp/FsAutoComplete`
+2. Build the FSAC solution and copy the dll output from the output log, it should be something like: `paket-files/github.com/fsharp/FsAutoComplete/src/FsAutoComplete/bin/Debug/netcoreapp2.1/fsautocomplete.dll`.  Nore `netcoreapp2.1` may be a different version.
+3. In the instance of VSCode that you have Ionide open, open settings `CMD ,`, and find the section `FSharp > Fsac: Net Core Dll Path` and paste the output you copied from step 3.
```

Comment:
I think we should add windows keyboard instruction too --> `Ctrl ,`

I guess `CMD` is for people using OSX.